kratos/python_scripts/read_materials_process.py

```diff
@@ -108,15 +132,15 @@ def _AssignPropertyBlock(self, data):
 
         # Set the CONSTITUTIVE_LAW for the current properties.
         if "Variables" in mat["constitutive_law"].keys(): #pass the list of variables when constructing the constitutive law
-           constitutive_law = self._GetItemFromModule( mat["constitutive_law"]["name"].GetString())(mat["constitutive_law"]["Variables"])
+           constitutive_law = self._GetVariable( mat["constitutive_law"]["name"].GetString())(mat["constitutive_law"]["Variables"])
```

Comment:
@RiccardoRossi I saw that you did this in the other branch
Why didn't you use `self._GetConstitutiveLaw` here too?

Reply:
to be honest i don't remember